Use AWS SES instead of SMTP

jsf9k opened the pull request:
Use AWS SES to send email messages (via [boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html)) instead of connecting directly to an SMTP server.

jsf9k committed:
Bumped version from 1.0.9 to 1.0.10

jsf9k committed:
Add .python-version to .gitignore

jsf9k committed:
Bumped version from 1.0.10 to 1.1.0

jsf9k committed:
Add boto3 as a dependency

jsf9k committed:
Use AWS SES and boto3 to send emails instead of SMTP

In other words, use boto3 to send emails via AWS SES as intended
instead of sending emails by connecting directly to AWS' SMTP server.

dav3r commented on cyhy/mailer/cli.py:
```diff
@@ -314,13 +273,26 @@ def get_federal_cyhy_requests(db):
 
-def send_message(mail_server, message, counter=None):
+class UnableToSendError(Exception):
+    """Raised when there an error is encountered when attempting to send
```
Minor typo in comment; remove `there`:   Raised when ~~there~~ an error is encountered...

jsf9k replied:
Fixed in d5794dd.

dav3r reviewed (approved):
Way to simplify! 👍

jsf9k committed:
Remove SMTP creds from docker-compose.yml file

jsf9k committed:
Fix a typo that @dav3r pointed out

jsf9k committed:
Add aws_config as a secret

This is to make sure we are talking to the correct region when sending
emails via SES.

jsf9k committed:
Add CyHy-only and BOD-only versions of docker-compose.yml

jsf9k committed:
Merge branch 'develop' into improvement/allow_use_of_ses

jsf9k committed:
Merge branch 'develop' into improvement/allow_use_of_ses

jsf9k committed:
Make shellcheck happy

jsf9k committed:
Add TravisCI config code to run shellcheck on shell scripts

jsf9k committed:
Make shellcheck happy and shellcheck the travis scripts too

jsf9k committed:
Fix a double-quoting bug I introduced

jsf9k committed:
Fix another bug that I introduced a few commits back

jsf9k committed:
Update Github URL in Dockerfile to use cisagov instead of dhs-ncats

jsf9k committed:
Update project URL in setup.py

jsf9k committed:
Update references to dhs-ncats so they point to cisagov instead

jsf9k commented:
@cisagov/team-ois Please review so we can get this old PR out the door. :older_man:

jsf9k committed:
Update agency name in setup.py

jsf9k committed:
Update NCATS URL
